### DISCUSSION TITLE
Please add one game to the compatibility list

### DISCUSSION THREAD

**Discussion by EduardoDuran0502** [skyline-emu/skyline#1533] [Game Requests]
could you please add the game Sword Art Online Hollow Realization to compatibility?, it's a great game, its graphics are similar to those of One piece, I wouldn't see any problem

**Comment by K11MCH1** [comment]
If you own it, you can test it and post it yourself.
If it is not on the list, it is because no one who owns it has not tested it yet, or posted it yet.

**Reply by EduardoDuran0502** [reply]
Sorry, but, How can I post it?, Im going to tested and i share my experience, thanks for the feedback

**Reply by Crytonics** [reply]
https://github.com/skyline-emu/skyline-games-list/issues
Just click on "New issue" and it will open template for game what to send, etc...
